build_aarch64_wheel.py

```diff
@@ -407,12 +411,13 @@ def configure_system(host: RemoteHost, *,
     if not use_conda:
         host.run_cmd("sudo apt-get install -y python3-dev python3-yaml python3-setuptools python3-wheel python3-pip")
     host.run_cmd("pip3 install dataclasses typing-extensions")
-    # Install and switch to gcc-8 on Ubuntu-18.04
-    if not host.using_docker() and host.ami == ubuntu18_04_ami and compiler == 'gcc-8':
-        host.run_cmd("sudo apt-get install -y g++-8 gfortran-8")
-        host.run_cmd("sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 100")
-        host.run_cmd("sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-8 100")
-        host.run_cmd("sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-8 100")
```

Comment:
Sorry, but why are you deleting this codepath? Please provide an explanation, why supporting ubuntu-18.04 is wrong here?

Reply:
Hi @malfet, thanks for the review! We need gcc-10 for Arm Compute Library multi-arch support, that's I replaced gcc-8 with gcc-10. But as you suggested, I will split this into incremental commits with more context.

Reply:
I think increasing min supported version to gcc-10 requires some more indepth discussion. I.e. is it compatible with aarch64 manylinux2014 requirements as outlined in https://peps.python.org/pep-0599/